package.json

```diff
@@ -292,7 +292,7 @@
     "webpack-dev-server": "^4.15.2",
     "webpack-notifier": "1.15.0",
     "webpack-stats-plugin": "^1.1.3",
-    "xlsx": "^0.18.5",
+    "xlsx": "https://cdn.sheetjs.com/xlsx-0.20.2/xlsx-0.20.2.tgz",
```

Comment:
The package documentation indicates that the distributed versions on npm are out of date.
https://docs.sheetjs.com/docs/getting-started/installation/nodejs/#legacy-endpoints